src/apps/loan-list/materials/selectable-material/selectable-material.tsx

```diff
@@ -83,27 +90,26 @@ const SelectableMaterial: FC<SelectableMaterialProps & MaterialProps> = ({
           <div>
             {statusBadgeComponent}
             {statusMessageComponentMobile}
-            {openDetailsModal && (
-              <button
-                type="button"
-                // This is to handle focus when more items are loaded via pagination
-                // eslint-disable-next-line jsx-a11y/no-autofocus
-                autoFocus={disabled && focused}
-                className="list-reservation__note"
-                onClick={() => openDetailsModal(id)}
-                aria-label={
-                  title
-                    ? t("groupModalGoToMaterialAriaLabelText", {
-                        placeholders: { "@label": title }
-                      })
-                    : ""
-                }
-              >
-                {t("groupModalGoToMaterialText")}
-              </button>
+            {displayedMaterial && (
+              <ReservationInfo
+                reservationInfo={displayedMaterial}
+                showArrow={false}
+                showStatusCircleIcon={false}
+                reservationStatusClassNameOverride=""
+                openReservationDetailsModal={() => {}}
```

Comment:
Consider if we should make `openReservationDetailsModal` optional.

The current handler does not do anything. Using optional props seems like a pattern we have been using elsewhere.

Reply:
👍  Dunno why GH is showing this. I have corrected it earlier..